.zenodo.json

```diff
@@ -0,0 +1,51 @@
+{
+  "creators": [
+    {
+      "name": "Ehbrecht, Carsten",
+      "affiliation": "DKRZ"
+    },
+    {
+      "name": "Charette-Migneault, Francis",
+      "affiliation": "CRIM",
+      "orcid": "0000-0003-4862-3349"
+    },
+    {
+      "name": "Derue, François-Xavier",
+      "affiliation": "CRIM"
+    },
+    {
+      "name": "Smith, Trevor James",
+      "affiliation": "Ouranos",
+      "orcid": "0000-0001-5393-8359"
+    },
+    {
+      "name": "Byrns, David",
+      "affiliation": "Intact"
+    },
+    {
+      "name": "Perron, Louis-David",
+      "affiliation": "CRIM"
+    }
+  ],
+  "keywords": [
+    "security",
```

Comment:
Add `authentication` and `authorization` to be more specific. `Policy Enforcement Point` is also a good one.